Sources/Sentry/SentrySDK.m

```diff
@@ -214,8 +218,19 @@ + (void)startWithOptions:(SentryOptions *)options
 #endif // TARGET_OS_IOS && SENTRY_HAS_UIKIT
 
 #if SENTRY_TARGET_PROFILING_SUPPORTED
+        BOOL shouldStopLaunchProfile = YES;
+#    if SENTRY_HAS_UIKIT
```

Comment:
`m`: This code changes quite some logic. IMO, we should add tests for this or have to change existing tests.

Reply:
That's exactly what the changes here are for: https://github.com/getsentry/sentry-cocoa/pull/3736/files#diff-e97a92e3a9df5dec5f35a1ee0603359d5857c05051313cc2c55897e54e7da05bL798-R813

These changes change what the start timestamp will be for the profile. The tests were changed to account for that. They now ensure that if the profile is attached to an app start txn, that the profile start time matches that of the app start txn.

Reply:
I am sorry I missed these updates. My bad 👍